website/docs/faq.mdx

```diff
@@ -47,3 +47,22 @@
     <p>For details on the registry format, please refer to <a href="https://github.com/TabbyML/registry-tabby/blob/main/models.json">models.json</a></p>
   </div>
 </details>
+
+<details>
+  <summary>Does Tabby currently support arm64/v8 CPUs?</summary>
+  <div>
+    <p>No, we don't have an official fix <b>yet</b>, however we have an experimental <a href="https://github.com/TabbyML/tabby/pull/1022/files">fix</a> available</p>
+    <p>To use this fix:</p>
+    <ol>
+      <li>Clone the tabby repo</li>
+      <li>Build the docker image locally with <code>--build-arg ARCHTARGET=amd64</code></li>
+      <li>Run the docker image</li>
+    </ol>
+    <pre>
+      <code>git clone https://github.com/TabbyML/tabby.git</code>
+      <code>cd tabby</code>
+      <code>docker build -t tabby:latest --build-arg ARCHTARGET=amd64 .</code>
```

Comment:
Not sure if this is the correct command, cc @jasonharrison

Reply:
Hi @wsxiaoys and @ll931217 - thanks so much for documenting this.  

The command should be:

```bash
docker build -t tabby:latest --platform linux/arm64 .
```
https://docs.docker.com/build/building/multi-platform/#building-multi-platform-images

I successfully built a Docker image using this command as of https://github.com/TabbyML/tabby/commit/5fe0b0667afa07946e4cba1272b5c85ee03149d4 .
<img width="1507" alt="Screenshot 2023-12-19 at 22 59 11" src="https://github.com/TabbyML/tabby/assets/304383/034d3e11-08e5-4b04-b12f-734a2b1ca00c">

Reply:
Okay thanks @jasonharrison , let me update that quick